src/main/java/de/tum/in/www1/artemis/domain/Participation.java

```diff
@@ -179,7 +182,14 @@ public void setSubmissions(Set<Submission> submissions) {
         this.submissions = submissions;
     }
 
-    // jhipster-needle-entity-add-getters-setters - JHipster will add getters and setters here, do not remove
+    @JsonProperty("latestSubmissionDate")
```

Comment:
I don't like this approach. I think it is better, if we find the latest submission and send it to the client together with its result (if available). To make the json smaller, we could remove some unnecessary elements of the submission (e.g. the model of the model submission or the text of the text submission)

Reply:
see my updates on the comment below

Reply:
Despite your additional comments above, the field should still be removed: https://github.com/ls1intum/Artemis/commit/68f569b514b9ecc7901aeafef7d5901780cfc9b8